code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -910,14 +914,14 @@
 
 /datum/reagent/medicine/dimorlin/on_mob_metabolize(mob/living/L)
 	..()
-	ADD_TRAIT(L, TRAIT_ANALGESIA, type)
+	ADD_TRAIT(L, TRAIT_PAIN_RESIST, type)
```

Comment:
dimorlin is written to be a Strong Painkiller, which is why it uses analgesia instead of pain_resist.

Reply:
shit I thought that was one of the ones I added oops